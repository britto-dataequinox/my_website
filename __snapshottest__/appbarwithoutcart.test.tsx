import React from 'react';
import renderer from 'react-test-renderer';
import AppBarWithoutCart from '../components/Header/AppbarWithoutCart';

jest.mock('next/router', () => ({
  useRouter() {
    return {
      route: '/',
      pathName: '',
      query: '',
      asPath: '/',
      push: jest.fn(),
    };
  },
}));

test('appbarwithoutcart snap', () => {
  const tree = renderer.create(<AppBarWithoutCart />).toJSON();
  expect(tree).toMatchSnapshot();
});
