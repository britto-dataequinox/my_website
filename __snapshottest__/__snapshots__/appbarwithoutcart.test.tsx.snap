// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`appbarwithoutcart snap 1`] = `
<header
  className="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-rjnxjj-MuiPaper-root-MuiAppBar-root"
>
  <div
    className="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-hyum1k-MuiToolbar-root"
  >
    <div
      className="MuiTypography-root MuiTypography-h6 css-1bn9r4q-MuiTypography-root"
      onClick={[Function]}
    >
      Online Store
    </div>
  </div>
</header>
`;
